doc/code_snippets/snippets/config/instances.enabled/persistence_snapshot/config.yaml

```diff
@@ -0,0 +1,14 @@
+groups:
+  group001:
+    replicasets:
+      replicaset001:
+        instances:
+          instance001:
+            snapshot:
+              dir: 'var/lib/{{ instance_name }}/snapshots'
+              count: 10
+              by:
+                interval: 60
```

Comment:
Snapshotting is not a lightweight operation (it writes a lot to a disc) and doing it every minute seems unusual. Every hour is the default. How about 7200 (every two hours) for the example.

I also doubt about storing 10 snapshots. The default is 2. But why one may need 10 snapshots? I see no use case. I propose to set it to 1 or, maybe, 3 for the example.